startup/19-pseudomotors.py

```diff
@@ -0,0 +1,66 @@
+print(f'Loading {__file__}...')
+
+import math
+from ophyd import (Device, PVPositionerPC, EpicsMotor, Signal, EpicsSignal,
+                   EpicsSignalRO, Component as Cpt, FormattedComponent as FCpt,
+                   PseudoSingle, PseudoPositioner,
+                   )
+from ophyd.pseudopos import (real_position_argument,
+                             pseudo_position_argument)
+from hxntools.device import NamedDevice #qustion for Ken
```

Comment:
What was this question about?
```suggestion
from hxntools.device import NamedDevice  # question for Ken
```

Reply:
I don't actually know what the question for Ken was. This was carried over from the old master branch.